web/app/test/testHelpers.jsx

```diff
@@ -0,0 +1,21 @@
+import _ from 'lodash';
+import { createMemoryHistory } from 'history';
+import React from 'react';
+import { Route, Router } from 'react-router';
+
+export function printStack() {
+  let e = new Error('dummy');
+  let stack = e.stack.replace(/^[^(]+?[\n$]/gm, '')
+    .replace(/^\s+at\s+/gm, '')
+    .replace(/^Object.<anonymous>\s*\(/gm, '{anonymous}()@')
+    .split('\n');
+  console.log(stack);
```

Comment:
I'd vote to `console.warn` this -- make it really obvious. That will also fix the lint warning.

Reply:
Oh, sorry - I completely missed this feedback before the merge - I can resubmit another branch with this method added in. Either for this release or something to be saved for the next one?

Reply:
No prob -- I'm all good with removing the method entirely, since it was unused.